assets/components/paymentAmount/paymentAmount.jsx

```diff
@@ -0,0 +1,32 @@
+// @flow
```

Comment:
Can you please add a screenshot of this component. It is easy to understand from the image if it is similar to another component.